### PR TITLE
feat: sort aexn transfers by txi_idx

### PR DESCRIPTION
**Pull request by sborrazas**

### CODE REVIEW

File: priv/migrations/20240910120001_restructure_aexn_transfers.ex [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  require Model
+
+  @spec run(State.t(), boolean()) :: {:ok, non_neg_integer()}
+  def run(state, _from_start?) do
+    transfer_mutations =
+      state
+      |> Collection.stream(Model.AexnTransfer, nil)
+      |> Stream.filter(fn {_aexn_type, _from_pk, _txi, to_pk, _value, _log_idx} ->
+        is_binary(to_pk)
```
Comment (vatanasov) [inline priv/migrations/20240910120001_restructure_aexn_transfers.ex:19]: Why is this filter needed? Won't this filter out the `nil` AexnTransfers and not move them to the new format?

File: priv/migrations/20240910120001_restructure_aexn_transfers.ex [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    zipped_mutations
+    |> Stream.chunk_every(1_000)
+    |> Stream.map(fn chunk ->
+      {mutations, deletion_keys} = Enum.unzip(chunk)
+
+      _mutations = [
+        DeleteKeysMutation.new(%{table => deletion_keys}) | mutations
+      ]
+
+      # _state = State.commit_db(state, mutations)
```
Comment (vatanasov) [inline priv/migrations/20240910120001_restructure_aexn_transfers.ex:78]: This should probably be uncommented

File: lib/ae_mdw/activities.ex [hunk 3]
```diff
@@ -643,15 +643,15 @@ defmodule AeMdw.Activities do
...
+            {aexn_type, account_pk, @min_int, @min_int, @min_bin, @min_int},
+            {aexn_type, account_pk, @max_int, @max_int, @max_bin, @max_int}
           }
 
         {first_txi, last_txi} ->
           {
             # The nil here accomodates for the fact that the second pk might be nil (in case of burn event)
-            {aexn_type, account_pk, first_txi, nil, @min_int, @min_int},
-            {aexn_type, account_pk, last_txi, @max_bin, @max_int, @max_int}
+            {aexn_type, account_pk, first_txi, @min_int, @min_bin, @min_int},
```
[the remark this answers is not in the document]
Comment (sborrazas) [reply lib/ae_mdw/activities.ex:652]: Yeap, it won't matter. Removed comment, thanks

File: priv/migrations/20240910120001_restructure_aexn_transfers.ex (hunk 1 above)
Comment (sborrazas) [reply priv/migrations/20240910120001_restructure_aexn_transfers.ex]: You're right, added the condition

File: priv/migrations/20240910120001_restructure_aexn_transfers.ex (hunk 2 above)
Comment (sborrazas) [reply priv/migrations/20240910120001_restructure_aexn_transfers.ex]: Woops, done

File: priv/migrations/20240910120001_restructure_aexn_transfers.ex [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+    {:ok, transfers_length + rev_transfers_length + pair_transfers_length}
+  end
+
+  defp replace_keys(state, table, zipped_mutations) do
+    zipped_mutations
+    |> Stream.chunk_every(1_000)
+    |> Stream.map(fn chunk ->
+      {mutations, deletion_keys} = Enum.unzip(chunk)
+
+      _mutations = [
```
Comment (vatanasov) [inline priv/migrations/20240910120001_restructure_aexn_transfers.ex:74]: I think those are the mutations we want

Comment (sborrazas) [reply priv/migrations/20240910120001_restructure_aexn_transfers.ex]: My bad, done!

**Review by vatanasov — APPROVED** [review]

**Review by yaboiishere — APPROVED** [review]

**Pull request merged**